h5p-bildetema-topic-list/src/components/index.tsx

```diff
@@ -0,0 +1 @@
+export { default as Word } from "./Word/Word";
```

Comment:
What is the argument for barrel exporting components like this?

Reply:
It makes it possible to import multiple components from a single line like this:
import { ContentTable, DropdownMenu, Header, DownloadButton } from "../";

Reply:
Yes, that's true. At least in the past, it also exported all of the above even when importing just one of them. Do you know if this has changed?